db/migrate/20191111090601_create_articles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.string :content
+      t.integer :user_id
```

Comment:
Prefer using references to create your foreign keys and add the `foreign_key: true` option. This line could instead be:
```ruby
t.references :user, foreign_key: true
```
The `references` will automatically create index in this field which is necessary for fast lookup. And the `foreign_key: true` option sets the field up as a foreign_key to the users table and applies foreign key contraints.